app/controllers/prevalidateTrader/PrevalidateTraderResultsController.scala

```diff
@@ -58,10 +58,11 @@ class PrevalidateTraderResultsController @Inject()(
         )
 
         val enteredEPCs: Seq[String] = request.userAnswers.get(PrevalidateTraderAddedValues).get
-        val ernToCheck: String = request.userAnswers.get(PrevalidateConsigneeTraderIdentificationPage).get
+        val ernToCheck: String = request.userAnswers.get(PrevalidateConsigneeTraderIdentificationPage).get.ern
+        val entityGroup: String = request.userAnswers.get(PrevalidateConsigneeTraderIdentificationPage).get.entityGroup.toString
```

Comment:
May be worth assigning `request.userAnswers.get(PrevalidateConsigneeTraderIdentificationPage).get` to a val to prevent 2 calls?

Reply:
Updated at https://github.com/hmrc/emcs-tfe-frontend/pull/169/commits/fca2432ee0ebe82b06fab8562deb606b17f3bff6